<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModifyEmployee</class>
 <widget class="QDialog" name="DialogModifyEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modification of Employee</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>361</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_Firstname">
      <property name="text">
       <string>Firstname</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Firstname"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Lastname">
      <property name="text">
       <string>Lastname : </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Lastname"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="text_ID">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_Types">
      <item>
       <property name="text">
        <string>Banker A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Banker B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Housing Insurer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Car Insurer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Life Insurer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Computer Scientist</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_ID">
      <property name="text">
       <string>ID : </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
